scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/core/properties/PASchedulerProperties.java

```diff
@@ -268,6 +271,12 @@ negative number indicates that session is infinite (value specified in second) *
     // Define the logging format pattern of tasks on the scheduler
     SCHEDULER_JOB_LOGS_PATTERN("pa.scheduler.job.task.output.logs.pattern", PropertyType.STRING),
 
+    SCHEDULER_DB_SIZE_MONITORING("pa.scheduler.db.size.monitoring", PropertyType.BOOLEAN),
```

Comment:
thanks, it's fixed. I forgot to remove the unused properties about this monitoring.